src/app/ubs/ubs-admin/components/ubs-admin-tariffs/ubs-admin-tariffs-location-dashboard.component.ts

```diff
@@ -133,6 +129,11 @@ export class UbsAdminTariffsLocationDashboardComponent implements OnInit, AfterV
     this.changeDetectorRef.detectChanges();
   }
 
+  public setDefaultStateValue(): void {
+    this.state.setValue('ACTIVE');
+    Object.assign(this.filterData, { status: 'ACTIVE' });
+  }
```

Comment:
You don't need to use setValue, just put 'ACTIVE' there when initializing form.

Reply:
I also think you should change state to status wherever it is.

Reply:
Thanks, it's good tips. 
Concerning changing state to status. 
I use key 'status' in filterData because it's required parameter name for  request. 
I think that it's better to clarify in Viktoriya whether it's should be changed. In case it is necessary, this changes should be in each component where we use 'state' instead of 'status'. 
I guess it's better to create new task card for this purpose. It is out of scope of this task.